docpages/example_programs/misc/using-emoji.md

```diff
@@ -0,0 +1,10 @@
+\page using-emoji Using Emoji
+
+Need your bot to use an emoji?
+Then the `dpp::emoji` class is just for you. It can store either a custom or a unicode emoji's name and ID. To use one in a message, you use its mention. To react, there's other kind of format: if you want to use an animated one, it's a:<name>:id, if it's static — <name>:id, if that's a unicode one — the corresponding character. Here, <name> represents the name of the emoji. In select menus, it's the emoji object itself. Here's how to use them: 
```

Comment:
This doesn't really make sense to be on the next line. It's part of the same paragraph.

Again, I think you still need to reword this as it's still making it sound like you should use dpp::emoji when using unicode emojis.

I would start it off with `Emojis are formed in two different, dpp::emoji and dpp::unicode_emoji. dpp::emoji is more commonly used for custom emojis, whereas dpp::unicode_emoji is used for the default emojis. Using dpp::unicode_emoji is pretty simple, but dpp::emoji is a bit more than that.`

Then you could have another paragraph where you start talking about how you add them in a message and then cut your examples into two, with an example here showing only that.

Then do a reactions part, with another example here with only reactions.

Then do the select menus part here.

Have a look at something like https://dpp.dev/setting_status.html, I seperate the examples into two, first example shows "hey, this is how you simply set the status", I then talk about doing it every x seconds and then show an example for that.

Also, `there's other kind of format` is incorrect, it should be `Reactions are a bit more complicated. For example, if you want to use an animated one.....`

Reply:
I don't like this either @Jaskowicz1 

dpp::unicode_emoji is not a class but a namespace, it's very different from dpp::emoji
The two should not be equated like that

Reply:
Hm true @Mishura4

The current text definitely needs to be split off, allowing three different examples

Reply:
What did I miss 💀

Reply:
> This doesn't really make sense to be on the next line. It's part of the same paragraph...
Allow me to yoink that